wiki/Game_Modes/osu!mania/en.md

```diff
@@ -1,447 +1,302 @@
+[osu!mania key layout2 image]: /wiki/shared/Mania_key_layouts2.jpg "osu!mania key layout (Co-Op)"
+[osu!mania editor image]: /wiki/shared/Editor_Mania.png "osu!mania-specific Editor"
+[osu!mania SSD image]: /wiki/shared/Song_Setup_Difficulty_Mania.jpg.jpg "osu!mania-specific Difficulty setup.jpg"
+[Options keyboard image]: /wiki/shared/Options_keyboard.jpg "Options Input icon, Keyboard section"
+
+<!-- Title -->
+
 # osu!mania
 
-![Gameplay of osu!mania](img/intro.jpg)
+**[Click to return to Game Modes page][Game_Modes wikilink]**
+
+![Gameplay of osu!mania][osu!mania Interface image]
+
+This mode has been widely used in almost all of the major rhythm games.
+It require good hand and/or leg coordination where the notes (with their quantity depending on BPM and difficulty) move on a conveyer.
+With this, the player will have to press the correct key for that specific note in time.
+Though the game mode was originally made as an imitation of the _Beatmania_ style of playing, osu!mania allows changing the number of keys or vertically flipping the orientation of the playfield (this means that it can be made to look more like _Guitar Hero_ \[5 keys\] or _Dance Dance Revolution_ \[4 keys\] and so on).
+
+It is somewhat similar to [osu!taiko][osu!taiko wikilink], but with more buttons notes moving vertically rather than moving horizontally.
+
+## ![osu!mania icon][osu!mania icon link] Gameplay explanation
+
+### Song Selection
+
+To access the osu!mania game mode, press `Ctrl`+`4` at the same time.
+
+Alternatively, click on the `Mode` button and select `osu!mania`.
+
+#### Keys and Judgement
+
+In the song selection screen, the number next to the _K_ shows how many keys will be used for the play.
+The beatmap will be played with looser judgement if "↓" symbol was appended at the end of _K_.
+
+For example, _4K↓_ means that it will be played with four(4) keys with a looser judgement timing than usual.
+
+Do note that the judgement of a beatmap is determined automatically.
+
+#### osu!specific beatmaps and conversion from osu! beatmap difference
+
+When converting a non-specific beatmap, the default key range would be around 4 to 7 keys.
+
+With the [xK][xK wikilink] game modifier, the player can manually set the number of keys from 1 to a total of 9 keys with a score multiplier reduction.
+However, this game modifier will not work with osu!mania-specific beatmaps.
+
+With the [Co-Op][Co-Op wikilink] game modifier, the stage will be split in two.
+This will use Co-Op's control scheme and will allows the player to play from 2 to a total of 18 keys with a score multiplier reduction.
+
+Do note that while the preset number of keys will not be doubled on osu!mania-specific beatmaps, it will split the stage in two (left stage priority if odd-numbered), use Co-Op's control scheme, and no score multiplier reduction.
+
+#### Speed Change
+
+![Speed Change value is located at the top-right][osu!mania SC image]
```

Comment:
- lol, please update this to use the default skin (this used my dummy user skin at the time)
- also, maybe different font-color (I chose that colour because white would have mixed with the other text)?
- please use _that_ specific mania beatmap when taking the screenshot (it has some meaning as it _is_ the first mania beatmap)
  - this one https://osu.ppy.sh/s/63089